edit-post/store/index.js

```diff
@@ -10,6 +15,7 @@ import reducer from './reducer';
 import enhanceWithBrowserSize from './mobile';
 import applyMiddlewares from './middlewares';
 import { BREAK_MEDIUM } from './constants';
+import { hasFixedToolbar } from './selectors';
```

Comment:
How do we decide to expose selectors or not, this one, for example, I'm fine keeping it as an internal.

If we're ok to accept the burden of maintaining BC for all selectors, I'd be open to consider exposing all selectors by default though to remove the need of `react-redux` entirely.

Reply:
>If we're ok to accept the burden of maintaining BC for all selectors, I'd be open to consider exposing all selectors by default though to remove the need of `react-redux` entirely.

Per discussion in today's JS chat, I think this is the direction I'd like to head toward. I'm not so inclined to fragmenting the data approach between first-party privileged access and third-party "hopefully it works, with what little data we grant access to".